### PR TITLE
Kubient adapter

### PR DESCRIPTION
**Pull request by mapcuk**
Opening pull request to add Kubient adapter to a prebid server.

### CODE REVIEW

**Comment by hhhjort** [comment]
It is a very simple adapter, but may be worth considering testing failure cases ... perhaps a bad response from your bidder?

**Comment by mapcuk** [comment]
> It is a very simple adapter, but may be worth considering testing failure cases ... perhaps a bad response from your bidder?

@hhhjort Yeah, it's quite simple as Kubient supports OpenRTB and doesn't require any extra special fields.
I'll add unit test for bad response. Anything else which should I care about?

**Comment by hhhjort** [comment]
Looks good otherwise to me. :)

File: usersync/usersyncers/syncer_test.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func TestNewSyncerMap(t *testing.T) {
 	}
 
 	adaptersWithoutSyncers := map[openrtb_ext.BidderName]bool{
-		openrtb_ext.BidderTappx: true,
+		openrtb_ext.BidderTappx:   true,
```
Comment (hhhjort) [inline usersync/usersyncers/syncer_test.go]: An extra space got in here somehow ... otherwise looks good.

**Comment by mapcuk** [comment]
@hhhjort I've added tests for bad response and finally tests passed.
Please check.

File: usersync/usersyncers/syncer_test.go (hunk 1 above)
Comment (mapcuk) [reply usersync/usersyncers/syncer_test.go]: Those lines were aligned by `gofmt -w usersync/usersyncers/syncer_test.go`

But when I remove spaces, **validate.sh** doesn't pass:
>  ./validate.sh --nofmt --cov --race 10
gofmt needs to be run, 1 files have issues.  Below is a list of files to review:
usersync/usersyncers/syncer_test.go

So Travis CI would not pass.

Comment (hhhjort) [reply usersync/usersyncers/syncer_test.go]: Odd, ow well, not worth wasting too much time over.

**Comment by mansinahar** [comment]
@mapcuk PR looks pretty straightforward to me but I see that it doesn't include usersync capability.  Does Kubient not support it? Just want to make sure that this isn't something that was mistakenly missed in the PR.

**Comment by mapcuk** [comment]
> @mapcuk PR looks pretty straightforward to me but I see that it doesn't include usersync capability. Does Kubient not support it? Just want to make sure that this isn't something that was mistakenly missed in the PR.

@mahsinahar It is not missed. Kubient doesn't support usersync by now.

**Comment by mansinahar** [comment]
Cool. Thanks for the confirmation @mapcuk !

**Pull request merged**